pom.xml

```diff
@@ -117,6 +117,7 @@
 			<version>3.3</version>
 		</dependency>
 
+<!--
```

Comment:
Same as above -- that's my intermediate scratch - I will remove once I remove the WIP from the pull request